definitions/npm/jest_v14.x.x/flow_>=v0.23.x/jest_v14.x.x.js

```diff
@@ -65,8 +97,16 @@ declare var jest: {
   setMock(moduleName: string, moduleExports: any): void;
   unmock(moduleName: string): void;
   useFakeTimers(): void;
+  useRealTimers(): void;
 }
 
 declare var jasmine: {
   DEFAULT_TIMEOUT_INTERVAL: number;
+  any(val: mixed): void;
```

Comment:
Nope; looks like it actually returns an "Any" object with properties `asymmetricMatch` and `jasmineToString`:

https://github.com/jasmine/jasmine/blob/f196faf0bbec55b2b912644f8b513214d6000980/src/core/asymmetric_equality/Any.js

I copied the definition over from jasmine_v2.4.x.js; do you think we should a definition for [Jasmine's "asymmetricMatch" objects](http://jasmine.github.io/2.4/introduction.html#section-Custom_asymmetric_equality_tester)?